docs/source/project_documentation/user_guide/model_definitions.md

```diff
@@ -60,8 +60,20 @@ triggers (`when`), and post-conditions (`then`) for the behavior.
     :lines: 6-38
     :emphasize-lines: 19, 21, 23, 25, 28, 30, 32
 ```
+### External Models
+
+An *External Model* is a `model` that does not include an *Acceptance Scenario*.  These types of `model`s are used to create simplified interfaces representing external components or actors.  A typical use case for External Model is to represent a user or external messenger that is not managed by AaC.  It allows you to quickly create `model`s without having to fully implement them.
+
+Bellow is an example External Model.
+
+```{eval-rst}
+.. literalinclude:: ../../../../python/features/alarm_clock/external.yaml
+    :language: yaml
+    :lines: 1-3
+```
+External Models can be referenced, created, and used the same as internal `model`s.
 
 ### Defining System State In Models
 
-*Pending an accurate model representation and sufficient implementation of state
-within models, this section will not be covered yet.*
+*Pending an accurate `model` representation and sufficient implementation of state
+within `model`s, this section will not be covered yet.*
```

Comment:
Similar to above, these aren't referencing the `model` field, but the model as a whole. 
```suggestion
*Pending an accurate model representation and sufficient implementation of state
within models, this section will not be covered yet.*
```